### DISCUSSION TITLE
Production ready?

### DISCUSSION THREAD

**Discussion by K2ouMais** [serversideup/docker-php#155] [Q&A] [answered]
Hello everyone,

it is said that I can use these containers in production, but these are running as the root user.

Am I missing something or does it has to work as root?

**Comment by jaydrogers** [comment, marked as answer]
Unfortunately the initial user needs to run as `root` to bring the processes up, then we run the actual php and web server process as `webroot` (this is how PHP-FPM wants it to be).

Good news is, when you're running any PHP application, it will run all applications as `webuser` inside the container.

Any other questions, let me know 👍
